packages/builder/src/pages/builder/apps/index.svelte

```diff
@@ -54,7 +54,11 @@
             </Layout>
             <ActionMenu align="right">
               <div slot="control" class="avatar">
-                <Avatar size="M" initials={$auth.initials} />
+                <Avatar
+                  size="M"
+                  initials={$auth.initials}
+                  url={$auth.user.pictureUrl}
```

Comment:
Is there always a pictureUrl key on the user object? Might want to add a wee `?.` there

Reply:
there might not be, it can evaluate to `undefined` and will be ignored by the `Avatar` component
https://github.com/Budibase/budibase/blob/feature/sso-sync-picture/packages/bbui/src/Avatar/Avatar.svelte#L18

Updated the screenshots to show what happens when a user doesn't have the `pictureUrl` attribute